omero/sysadmins/system-requirements.txt

```diff
@@ -106,3 +81,26 @@ then these temporary files are being written and read over the
 network. This can slow access down.  The :envvar:`OMERO_TEMPDIR`
 environment variable may used to set the directory used for temporary
 files, for example on fast local storage.
+
+Software
+--------
+
+Each component of the OMERO platform has a separate set of
+prerequisites. Where possible, we provide tips on getting started with
+each of these technologies, but we can only provide free support within
+limits.
+
+============== =============================== ======== ======== ======== ==========
+Package        OMERO.server                    Java     Python   Ice      PostgreSQL
+============== =============================== ======== ======== ======== ==========
+OMERO.importer Required                        Required
+OMERO.insight  Required                        Required
+OMERO.server                                   Required Required Required Required
+OMERO.web      Required                                 Required Required
+OMERO.py       Required for some functionality          Required Required
+OMERO.cpp      Required for some functionality                   Required
+============== =============================== ======== ======== ======== ==========
+
+Full details on which versions of these are supported for 5.1.x and how we
+intend to update these going forward is provided on the
+:doc:`version-requirements` page.
```

Comment:
Could it be reworded not to use `page`, since this isn't suitable for the PDF?  e.g.

```
  For full details on ... please see the :doc: section
```